_i18n/en/documentation/getting-started/home.md

```diff
@@ -0,0 +1,20 @@
+The Home screen, introduced in AntennaPod 3.0, brought about a new way to have a quick overview of your podcasts, start listening and perform a range of actions.
```

Comment:
I think I'd merge this paragraph with the next one. In any case, we typically don't mention when a feature is introduced so that (and 'brought about…') should be removed.